### PR TITLE
Removes specific names from clone pod reports

### PR DESCRIPTION
**Pull request by Poojawa**
:cl: Poojawa
tweak: Clone pods no longer announce the name of the clone
/:cl:

dat meta game busting.

### CODE REVIEW

**Review by deathride58 — APPROVED** [review]
Requiring approval from at least one other maintainer for this to make it in.
Personally, I think it'll help add a small layer of paranoia and help encourage medical personnel to actually keep an eye on the cloner. Antags will be able to autoclone without the Captain noticing, and instead of an autoclonable miner prompting a "oh, it was just a miner who died" response from medical, it'll prompt a "the round's quiet as fuck, who the fuck just died?" response.

**Comment by Poojawa** [comment]
Kev was ok with the change last I heard, so.

**Pull request merged**
